src/std_extensions/arithmetic/float_ops.rs

```diff
@@ -41,77 +36,60 @@ pub fn extension() -> Extension {
     );
 
     extension
-        .add_op_custom_sig_simple("feq".into(), "equality test".to_owned(), vec![], fcmp_sig)
+        .add_op_type_scheme_simple("feq".into(), "equality test".to_owned(), fcmp_sig())
         .unwrap();
     extension
-        .add_op_custom_sig_simple("fne".into(), "inequality test".to_owned(), vec![], fcmp_sig)
+        .add_op_type_scheme_simple("fne".into(), "inequality test".to_owned(), fcmp_sig())
         .unwrap();
     extension
-        .add_op_custom_sig_simple("flt".into(), "\"less than\"".to_owned(), vec![], fcmp_sig)
+        .add_op_type_scheme_simple("flt".into(), "\"less than\"".to_owned(), fcmp_sig())
```

Comment:
@cqc-alec ?

Reply:
The name of the operation is "less than" in quotes because the convention for operation names is that they should be nouns (addition, maximum etc), and "less than" without quotes would be inconsistent with that -- it would have to be "less-than comparator" or somthing, which is more verbose. But it is rather awkward I agree; we could remove them if you like.